sentry-rails/spec/sentry/rails_spec.rb

```diff
@@ -14,12 +14,6 @@
       make_basic_app
     end
 
-    after do
-      # We need to cleanup Rails.logger because after https://github.com/rails/rails/pull/49417
-      # Rails.logger could get recreated with the previous logger value and become deeply nested broadcast logger
-      Rails.logger = nil
-    end
```

Comment:
The behaviour has been addressed in https://github.com/rails/rails/pull/49462, so we don't need this anymore.